gui/js/tray.window.js

```diff
@@ -155,7 +155,8 @@ module.exports = class TrayWM extends WindowManager {
       'auto-launcher': (event, enabled) => autoLaunch.setEnabled(enabled),
       'close-app': () => this.desktop.stopSync().then(() => this.app.quit()),
       'open-file': (event, path) => this.openPath(path),
-      'unlink-cozy': this.onUnlink
+      'unlink-cozy': this.onUnlink,
+      'manual-start-sync': () => this.desktop.synchronize('full')
```

Comment:
I really don't think that `this.desktop.synchronize('full')` is what we want to do. It does a lot of things, including taking changes from pouchdb and trying to apply them, ie after the user has started a manual sync, we may have several files that can be downloaded/uploaded at the same time and it can lead to a lot of bugs.

Reply:
I though that `this.desktop.synchronize` was the counterpart of `this.desktop.stopSync()`.
As said there are so much things called `sync` that it is hard to find the one that start a sync workflow: looking on the server if there is something new, looking on the local system if there is something new, apply and confirm to the end-user that the desktop client and the remote server are synced.

Reply:
Can you help me by indicating the function of the workflow that you think is appropriate?

Reply:
The trick is that I don't think we have a function that is appropriate. Maybe you can do some pair programming to find what we can do here.